loopy/symbolic.py

```diff
@@ -1621,7 +1621,8 @@ def qpolynomial_from_expr(space, expr):
 
 
 # {{{ simplify using aff
-@memoize_method
+
+@memoize_on_first_arg
```

Comment:
Thanks

Reply:
Np. It's mostly equivalent, however I find that "method" looks weird on things that aren't a method. :)